### ISSUE TITLE
Full Screen Switch Monitors Crashes it

### ISSUE THREAD

**Issue by cnnradams** [AstechzGO/Luminescent#81]
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x501b3b74, pid=5336, tid=4100
#
# JRE version: Java(TM) SE Runtime Environment (8.0_66-b18) (build 1.8.0_66-b18)
# Java VM: Java HotSpot(TM) Client VM (25.66-b18 mixed mode windows-x86 )
# Problematic frame:
# C  [glfw32.dll+0x3b74]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  T H R E A D  ---------------

Current thread (0x00a8c000):  JavaThread "main" [_thread_in_native, id=4100, stack(0x009e0000,0x00a30000)]

siginfo: ExceptionCode=0xc0000005, reading address 0x00000000

Registers:
EAX=0x00000000, EBX=0x00000000, ECX=0x00000000, EDX=0x00000000
ESP=0x00a2f39c, EBP=0x00a2f3a0, ESI=0x19fb8b40, EDI=0x00a8c000
EIP=0x501b3b74, EFLAGS=0x00210287

Top of Stack: (sp=0x00a2f39c)
0x00a2f39c:   00a2f3d0 00a2f3e8 52c1f538 19fb8b40
0x00a2f3ac:   027f1139 00a8c140 00a2f3d0 501b3b00
0x00a2f3bc:   00000000 19fb8b40 00000000 00000000
0x00a2f3cc:   1515c520 09d357e0 00a2f3e8 00a2f414
0x00a2f3dc:   025e4094 501b2ab0 00000000 00a2f434
0x00a2f3ec:   025e4134 501b3b00 00000000 19fb8b40
0x00a2f3fc:   00000000 025e4134 19fb8b40 00000000
0x00a2f40c:   501b3b00 00000000 00a2f414 15159d14 

Instructions: (pc=0x501b3b74)
0x501b3b54:   56 0f 44 c1 a3 70 c1 1d 50 e8 9e 1e 00 00 83 c4
0x501b3b64:   04 b9 6c c1 1d 50 39 35 6c c1 1d 50 74 06 8b 09
0x501b3b74:   39 31 75 fa 8b 06 56 89 01 e8 d2 d6 00 00 83 c4
0x501b3b84:   04 5e 5d c3 cc cc cc cc cc cc cc cc 55 8b ec 8b 


Register to memory mapping:

EAX=0x00000000 is an unknown value
EBX=0x00000000 is an unknown value
ECX=0x00000000 is an unknown value
EDX=0x00000000 is an unknown value
ESP=0x00a2f39c is pointing into the stack for thread: 0x00a8c000
EBP=0x00a2f3a0 is pointing into the stack for thread: 0x00a8c000
ESI=0x19fb8b40 is an unknown value
EDI=0x00a8c000 is a thread


Stack: [0x009e0000,0x00a30000],  sp=0x00a2f39c,  free space=316k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [glfw32.dll+0x3b74]
C  [lwjgl32.dll+0x2f538]
j  org.lwjgl.glfw.GLFW.glfwDestroyWindow(J)V+20
j  astechzgo.luminescent.utils.DisplayUtils.nextMonitor()V+320
J 735 C1 astechzgo.luminescent.main.Luminescent.Tick()V (1731 bytes) @ 0x027c76cc [0x027c4f10+0x27bc]
J 890 C1 astechzgo.luminescent.main.Main.Tick()V (16 bytes) @ 0x027f42f0 [0x027f4190+0x160]
j  astechzgo.luminescent.main.Main.Run()V+8
j  astechzgo.luminescent.main.Main.main([Ljava/lang/String;)V+10
v  ~StubRoutines::call_stub
V  [jvm.dll+0x158ba5]
V  [jvm.dll+0x21e12e]
V  [jvm.dll+0x158c3e]
V  [jvm.dll+0xdae57]
V  [jvm.dll+0xe33ef]
C  [javaw.exe+0x229a]
C  [javaw.exe+0xaebf]
C  [javaw.exe+0xaf49]
C  [KERNEL32.DLL+0x13744]
C  [ntdll.dll+0x59cd4]
C  [ntdll.dll+0x59c9f]
C  0x00000000

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
J 877  org.lwjgl.system.JNI.invokePV(JJ)V (0 bytes) @ 0x027f10f5 [0x027f10a0+0x55]
j  org.lwjgl.glfw.GLFW.glfwDestroyWindow(J)V+20
j  astechzgo.luminescent.utils.DisplayUtils.nextMonitor()V+320
J 735 C1 astechzgo.luminescent.main.Luminescent.Tick()V (1731 bytes) @ 0x027c76cc [0x027c4f10+0x27bc]
J 890 C1 astechzgo.luminescent.main.Main.Tick()V (16 bytes) @ 0x027f42f0 [0x027f4190+0x160]
j  astechzgo.luminescent.main.Main.Run()V+8
j  astechzgo.luminescent.main.Main.main([Ljava/lang/String;)V+10
v  ~StubRoutines::call_stub

---------------  P R O C E S S  ---------------

Java Threads: ( => current thread )
  0x154aec00 JavaThread "Timer-0" [_thread_blocked, id=3052, stack(0x1c0b0000,0x1c100000)]
  0x154ca400 JavaThread "Thread-5" [_thread_blocked, id=1008, stack(0x1b6b0000,0x1b700000)]
  0x154c7800 JavaThread "Thread-4" [_thread_blocked, id=360, stack(0x1b620000,0x1b670000)]
  0x00f22800 JavaThread "AWT-Windows" daemon [_thread_in_native, id=3724, stack(0x14f70000,0x14fc0000)]
  0x00f21000 JavaThread "Java2D Disposer" daemon [_thread_blocked, id=3268, stack(0x14e50000,0x14ea0000)]
  0x00dc9800 JavaThread "Service Thread" daemon [_thread_blocked, id=6788, stack(0x14c50000,0x14ca0000)]
  0x00dc3000 JavaThread "C1 CompilerThread0" daemon [_thread_blocked, id=2524, stack(0x14bc0000,0x14c10000)]
  0x00dc1800 JavaThread "Attach Listener" daemon [_thread_blocked, id=3972, stack(0x14b30000,0x14b80000)]
  0x00dcc400 JavaThread "Signal Dispatcher" daemon [_thread_blocked, id=1272, stack(0x14aa0000,0x14af0000)]
  0x00da7400 JavaThread "Finalizer" daemon [_thread_blocked, id=1276, stack(0x01070000,0x010c0000)]
  0x00da2800 JavaThread "Reference Handler" daemon [_thread_blocked, id=164, stack(0x00fe0000,0x01030000)]
=>0x00a8c000 JavaThread "main" [_thread_in_native, id=4100, stack(0x009e0000,0x00a30000)]

Other Threads:
  0x00b8bc00 VMThread [stack: 0x00d50000,0x00da0000] [id=7660]
  0x00df3400 WatcherThread [stack: 0x14ce0000,0x14d30000] [id=8148]

VM state:not at safepoint (normal execution)

VM Mutex/Monitor currently owned by a thread: None

Heap:
 def new generation   total 4928K, used 1103K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K,  24% used [0x04600000, 0x0470b0f8, 0x04a50000)
  from space 512K,   6% used [0x04ad0000, 0x04ad8df0, 0x04b50000)
  to   space 512K,   0% used [0x04a50000, 0x04a50000, 0x04ad0000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a7e0, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K

Card table byte_map: [0x00c60000,0x00cf0000] byte_map_base: 0x00c3d000

Polling page: 0x005b0000

CodeCache: size=32768Kb used=2161Kb max_used=2161Kb free=30606Kb
 bounds [0x025e0000, 0x02800000, 0x045e0000]
 total_blobs=1365 nmethods=901 adapters=396
 compilation: enabled

Compilation events (10 events):
Event: 74.806 Thread 0x00dc3000  899             java.util.HashSet::add (20 bytes)
Event: 74.806 Thread 0x00dc3000 nmethod 899 0x027fb748 code [0x027fb850, 0x027fb8fc]
Event: 74.806 Thread 0x00dc3000  900             org.lwjgl.system.APIBuffer::address (8 bytes)
Event: 74.806 Thread 0x00dc3000 nmethod 900 0x027fb948 code [0x027fba40, 0x027fbad0]
Event: 74.806 Thread 0x00dc3000  901             org.lwjgl.opengl.GL::getFunctionAddress (16 bytes)
Event: 74.806 Thread 0x00dc3000 nmethod 901 0x027fbb08 code [0x027fbc00, 0x027fbcac]
Event: 74.806 Thread 0x00dc3000  902             org.lwjgl.system.windows.WindowsLibrary::getFunctionAddress (9 bytes)
Event: 74.807 Thread 0x00dc3000 nmethod 902 0x027fbd08 code [0x027fbe30, 0x027fbff0]
Event: 74.807 Thread 0x00dc3000  903             org.lwjgl.system.windows.WinBase::GetProcAddress (34 bytes)
Event: 74.807 Thread 0x00dc3000 nmethod 903 0x027fc1c8 code [0x027fc2f0, 0x027fc490]

GC Heap History (10 events):
Event: 70.300 GC heap before
{Heap before GC invocations=172 (full 0):
 def new generation   total 4928K, used 4424K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K, 100% used [0x04600000, 0x04a50000, 0x04a50000)
  from space 512K,   1% used [0x04a50000, 0x04a52288, 0x04ad0000)
  to   space 512K,   0% used [0x04ad0000, 0x04ad0000, 0x04b50000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a760, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
Event: 70.302 GC heap after
Heap after GC invocations=173 (full 0):
 def new generation   total 4928K, used 5K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K,   0% used [0x04600000, 0x04600000, 0x04a50000)
  from space 512K,   1% used [0x04ad0000, 0x04ad16c8, 0x04b50000)
  to   space 512K,   0% used [0x04a50000, 0x04a50000, 0x04ad0000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a760, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
}
Event: 71.378 GC heap before
{Heap before GC invocations=173 (full 0):
 def new generation   total 4928K, used 4421K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K, 100% used [0x04600000, 0x04a50000, 0x04a50000)
  from space 512K,   1% used [0x04ad0000, 0x04ad16c8, 0x04b50000)
  to   space 512K,   0% used [0x04a50000, 0x04a50000, 0x04ad0000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a760, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
Event: 71.379 GC heap after
Heap after GC invocations=174 (full 0):
 def new generation   total 4928K, used 18K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K,   0% used [0x04600000, 0x04600000, 0x04a50000)
  from space 512K,   3% used [0x04a50000, 0x04a54978, 0x04ad0000)
  to   space 512K,   0% used [0x04ad0000, 0x04ad0000, 0x04b50000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a760, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
}
Event: 72.410 GC heap before
{Heap before GC invocations=174 (full 0):
 def new generation   total 4928K, used 4434K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K, 100% used [0x04600000, 0x04a50000, 0x04a50000)
  from space 512K,   3% used [0x04a50000, 0x04a54978, 0x04ad0000)
  to   space 512K,   0% used [0x04ad0000, 0x04ad0000, 0x04b50000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a760, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
Event: 72.411 GC heap after
Heap after GC invocations=175 (full 0):
 def new generation   total 4928K, used 18K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K,   0% used [0x04600000, 0x04600000, 0x04a50000)
  from space 512K,   3% used [0x04ad0000, 0x04ad4aa0, 0x04b50000)
  to   space 512K,   0% used [0x04a50000, 0x04a50000, 0x04ad0000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a760, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
}
Event: 73.226 GC heap before
{Heap before GC invocations=175 (full 0):
 def new generation   total 4928K, used 4434K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K, 100% used [0x04600000, 0x04a50000, 0x04a50000)
  from space 512K,   3% used [0x04ad0000, 0x04ad4aa0, 0x04b50000)
  to   space 512K,   0% used [0x04a50000, 0x04a50000, 0x04ad0000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a760, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
Event: 73.227 GC heap after
Heap after GC invocations=176 (full 0):
 def new generation   total 4928K, used 14K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K,   0% used [0x04600000, 0x04600000, 0x04a50000)
  from space 512K,   2% used [0x04a50000, 0x04a53b08, 0x04ad0000)
  to   space 512K,   0% used [0x04ad0000, 0x04ad0000, 0x04b50000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a7e0, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
}
Event: 74.143 GC heap before
{Heap before GC invocations=176 (full 0):
 def new generation   total 4928K, used 4430K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K, 100% used [0x04600000, 0x04a50000, 0x04a50000)
  from space 512K,   2% used [0x04a50000, 0x04a53b08, 0x04ad0000)
  to   space 512K,   0% used [0x04ad0000, 0x04ad0000, 0x04b50000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a7e0, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
Event: 74.144 GC heap after
Heap after GC invocations=177 (full 0):
 def new generation   total 4928K, used 35K [0x04600000, 0x04b50000, 0x09b50000)
  eden space 4416K,   0% used [0x04600000, 0x04600000, 0x04a50000)
  from space 512K,   6% used [0x04ad0000, 0x04ad8df0, 0x04b50000)
  to   space 512K,   0% used [0x04a50000, 0x04a50000, 0x04ad0000)
 tenured generation   total 10944K, used 8425K [0x09b50000, 0x0a600000, 0x14600000)
   the space 10944K,  76% used [0x09b50000, 0x0a38a7e0, 0x0a38a800, 0x0a600000)
 Metaspace       used 8185K, capacity 8373K, committed 8448K, reserved 8576K
}

Deoptimization events (0 events):
No events

Internal exceptions (10 events):
Event: 0.258 Thread 0x00a8c000 Exception <a 'java/security/PrivilegedActionException'> (0x046d01b8) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jvm.cpp, line 1386]
Event: 0.258 Thread 0x00a8c000 Exception <a 'java/security/PrivilegedActionException'> (0x046d03b8) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jvm.cpp, line 1386]
Event: 0.259 Thread 0x00a8c000 Exception <a 'java/security/PrivilegedActionException'> (0x046ea308) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jvm.cpp, line 1386]
Event: 0.259 Thread 0x00a8c000 Exception <a 'java/security/PrivilegedActionException'> (0x046ea508) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jvm.cpp, line 1386]
Event: 0.288 Thread 0x00a8c000 Exception <a 'java/io/FileNotFoundException'> (0x04724770) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jni.cpp, line 709]
Event: 0.332 Thread 0x00a8c000 Exception <a 'java/lang/NoSuchFieldError': method resolution failed> (0x047df210) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\methodHandles.cpp, line 1146]
Event: 0.333 Thread 0x00a8c000 Exception <a 'java/lang/NoSuchFieldError': method resolution failed> (0x047eb770) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\methodHandles.cpp, line 1146]
Event: 0.420 Thread 0x00a8c000 Exception <a 'java/io/FileNotFoundException'> (0x04882ba8) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jni.cpp, line 709]
Event: 2.157 Thread 0x154c7800 Exception <a 'java/lang/InterruptedException': sleep interrupted> (0x046849e0) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jvm.cpp, line 3215]
Event: 2.557 Thread 0x154c7800 Exception <a 'java/lang/InterruptedException': sleep interrupted> (0x04698478) thrown at [C:\re\workspace\8-2-build-windows-i586-cygwin\jdk8u66\5298\hotspot\src\share\vm\prims\jvm.cpp, line 3215]

Events (10 events):
Event: 70.467 Executing VM operation: ThreadDump
Event: 70.467 Executing VM operation: ThreadDump done
Event: 71.377 Executing VM operation: GenCollectForAllocation
Event: 71.379 Executing VM operation: GenCollectForAllocation done
Event: 72.410 Executing VM operation: GenCollectForAllocation
Event: 72.411 Executing VM operation: GenCollectForAllocation done
Event: 73.226 Executing VM operation: GenCollectForAllocation
Event: 73.227 Executing VM operation: GenCollectForAllocation done
Event: 74.143 Executing VM operation: GenCollectForAllocation
Event: 74.144 Executing VM operation: GenCollectForAllocation done


Dynamic libraries:
0x011a0000 - 0x011d3000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\javaw.exe
0x775e0000 - 0x77759000     C:\WINDOWS\SYSTEM32\ntdll.dll
0x76fa0000 - 0x77090000     C:\WINDOWS\SYSTEM32\KERNEL32.DLL
0x76810000 - 0x76986000     C:\WINDOWS\SYSTEM32\KERNELBASE.dll
0x76190000 - 0x7620b000     C:\WINDOWS\SYSTEM32\ADVAPI32.dll
0x76e30000 - 0x76eee000     C:\WINDOWS\SYSTEM32\msvcrt.dll
0x77590000 - 0x775d3000     C:\WINDOWS\SYSTEM32\sechost.dll
0x76530000 - 0x765dc000     C:\WINDOWS\SYSTEM32\RPCRT4.dll
0x746f0000 - 0x7470e000     C:\WINDOWS\SYSTEM32\SspiCli.dll
0x746e0000 - 0x746ea000     C:\WINDOWS\SYSTEM32\CRYPTBASE.dll
0x74680000 - 0x746d9000     C:\WINDOWS\SYSTEM32\bcryptPrimitives.dll
0x75f30000 - 0x76070000     C:\WINDOWS\SYSTEM32\USER32.dll
0x75d40000 - 0x75e8d000     C:\WINDOWS\SYSTEM32\GDI32.dll
0x73aa0000 - 0x73ca9000     C:\WINDOWS\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.10240.16384_none_3bccb1ff6bcd1849\COMCTL32.dll
0x76210000 - 0x7623b000     C:\WINDOWS\SYSTEM32\IMM32.DLL
0x76070000 - 0x76190000     C:\WINDOWS\SYSTEM32\MSCTF.dll
0x52c70000 - 0x52d2f000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\msvcr100.dll
0x64840000 - 0x64c05000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\client\jvm.dll
0x770a0000 - 0x770a6000     C:\WINDOWS\SYSTEM32\PSAPI.DLL
0x6a060000 - 0x6a068000     C:\WINDOWS\SYSTEM32\WSOCK32.dll
0x73d10000 - 0x73d34000     C:\WINDOWS\SYSTEM32\WINMM.dll
0x76a00000 - 0x76a5c000     C:\WINDOWS\SYSTEM32\WS2_32.dll
0x77090000 - 0x77097000     C:\WINDOWS\SYSTEM32\NSI.dll
0x74670000 - 0x74678000     C:\WINDOWS\SYSTEM32\VERSION.dll
0x73ce0000 - 0x73d03000     C:\WINDOWS\SYSTEM32\WINMMBASE.dll
0x76b60000 - 0x76b96000     C:\WINDOWS\SYSTEM32\cfgmgr32.dll
0x73cb0000 - 0x73cd1000     C:\WINDOWS\SYSTEM32\DEVOBJ.dll
0x715d0000 - 0x715dc000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\verify.dll
0x6b7f0000 - 0x6b811000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\java.dll
0x713f0000 - 0x71403000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\zip.dll
0x74890000 - 0x75c4e000     C:\WINDOWS\SYSTEM32\SHELL32.dll
0x770b0000 - 0x7758c000     C:\WINDOWS\SYSTEM32\windows.storage.dll
0x76ba0000 - 0x76d5a000     C:\WINDOWS\SYSTEM32\combase.dll
0x76b10000 - 0x76b54000     C:\WINDOWS\SYSTEM32\shlwapi.dll
0x769e0000 - 0x769ec000     C:\WINDOWS\SYSTEM32\kernel.appcore.dll
0x76250000 - 0x762dd000     C:\WINDOWS\SYSTEM32\shcore.dll
0x76990000 - 0x769d4000     C:\WINDOWS\SYSTEM32\powrprof.dll
0x769f0000 - 0x769ff000     C:\WINDOWS\SYSTEM32\profapi.dll
0x6b7d0000 - 0x6b7e6000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\net.dll
0x739a0000 - 0x739ee000     C:\WINDOWS\system32\mswsock.dll
0x6b7c0000 - 0x6b7cf000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\nio.dll
0x501f0000 - 0x50335000     C:\Program Files (x86)\Java\jre1.8.0_66\bin\awt.dll
0x76490000 - 0x76522000     C:\WINDOWS\SYSTEM32\OLEAUT32.dll
0x61da0000 - 0x61e31000     C:\WINDOWS\system32\apphelp.dll
0x734b0000 - 0x73525000     C:\WINDOWS\system32\uxtheme.dll
0x73490000 - 0x734ad000     C:\WINDOWS\system32\dwmapi.dll
0x75c50000 - 0x75d3a000     C:\WINDOWS\SYSTEM32\ole32.dll
0x52bf0000 - 0x52c6d000     C:\Users\cnnr2\Desktop\lwjgl\native\lwjgl32.dll
0x52b60000 - 0x52b95000     C:\Users\cnnr2\Desktop\lwjgl\native\jemalloc32.dll
0x501b0000 - 0x501e4000     C:\Users\cnnr2\Desktop\lwjgl\native\glfw32.dll
0x6a190000 - 0x6a270000     C:\WINDOWS\SYSTEM32\opengl32.dll
0x6a160000 - 0x6a185000     C:\WINDOWS\SYSTEM32\GLU32.dll
0x6a070000 - 0x6a15b000     C:\WINDOWS\SYSTEM32\DDRAW.dll
0x69a90000 - 0x69a97000     C:\WINDOWS\SYSTEM32\DCIMAN32.dll
0x50190000 - 0x501a5000     C:\WINDOWS\SYSTEM32\atiglpxx.dll
0x68050000 - 0x6990a000     C:\WINDOWS\SYSTEM32\atioglxx.dll
0x762e0000 - 0x76485000     C:\WINDOWS\SYSTEM32\SETUPAPI.dll
0x77f00000 - 0x77ff1000     C:\WINDOWS\SYSTEM32\atiadlxy.dll
0x73e80000 - 0x73e99000     C:\WINDOWS\SYSTEM32\USERENV.dll
0x73da0000 - 0x73dd0000     C:\WINDOWS\SYSTEM32\IPHLPAPI.DLL
0x717a0000 - 0x718e2000     C:\WINDOWS\SYSTEM32\PROPSYS.dll
0x73ea0000 - 0x73eaf000     C:\WINDOWS\SYSTEM32\WTSAPI32.dll
0x73d40000 - 0x73d48000     C:\WINDOWS\SYSTEM32\WINNSI.DLL
0x76f50000 - 0x76f92000     C:\WINDOWS\SYSTEM32\WINTRUST.dll
0x76240000 - 0x7624e000     C:\WINDOWS\SYSTEM32\MSASN1.dll
0x74710000 - 0x74885000     C:\WINDOWS\SYSTEM32\CRYPT32.dll
0x50160000 - 0x50187000     C:\WINDOWS\SYSTEM32\atigktxx.dll
0x73e60000 - 0x73e73000     C:\WINDOWS\SYSTEM32\CRYPTSP.dll
0x73e40000 - 0x73e5b000     C:\WINDOWS\SYSTEM32\bcrypt.dll
0x73e10000 - 0x73e3f000     C:\WINDOWS\system32\rsaenh.dll
0x73a10000 - 0x73a23000     C:\WINDOWS\SYSTEM32\dhcpcsvc6.DLL
0x739f0000 - 0x73a04000     C:\WINDOWS\SYSTEM32\dhcpcsvc.DLL
0x77e30000 - 0x77ef5000     C:\Users\cnnr2\Desktop\lwjgl\native\OpenAL32.dll
0x76a60000 - 0x76ae2000     C:\WINDOWS\SYSTEM32\clbcatq.dll
0x71610000 - 0x71664000     C:\WINDOWS\System32\MMDevApi.dll
0x6e010000 - 0x6e096000     C:\WINDOWS\SYSTEM32\dsound.dll
0x6cba0000 - 0x6cc08000     C:\WINDOWS\SYSTEM32\AUDIOSES.DLL
0x6da20000 - 0x6dae5000     C:\WINDOWS\SYSTEM32\wintypes.dll
0x50130000 - 0x50155000     C:\WINDOWS\SYSTEM32\DINPUT.DLL
0x65380000 - 0x6538a000     C:\WINDOWS\SYSTEM32\HID.DLL
0x6dd40000 - 0x6de7f000     C:\WINDOWS\SYSTEM32\dbghelp.dll

VM Arguments:
jvm_args: -Dorg.lwjgl.librarypath=C:/users/cnnr2/Desktop/Luminescent/libs/lwjgl/natives -Djava.library.path=C:\Users\cnnr2\Desktop\lwjgl\native -Dfile.encoding=Cp1252 
java_command: astechzgo.luminescent.main.Main
java_class_path (initial): C:\Users\cnnr2\Desktop\Luminescent\Luminescent\bin;C:\Users\cnnr2\Desktop\Luminescent\Luminescent\libs\lwjgl\lwjgl.jar;C:\Users\cnnr2\Desktop\Luminescent\Luminescent\libs\paulscode\SoundSystem.jar;C:\Users\cnnr2\Desktop\Luminescent\Luminescent\libs\paulscode\CodecMP3.jar;C:\Users\cnnr2\Desktop\Luminescent\Luminescent\libs\paulscode\LibraryLWJGL3OpenAL.jar
Launcher Type: SUN_STANDARD

Environment Variables:
PATH=C:/Program Files (x86)/Java/jre1.8.0_66/bin/client;C:/Program Files (x86)/Java/jre1.8.0_66/bin;C:/Program Files (x86)/Java/jre1.8.0_66/lib/i386;C:\ProgramData\Oracle\Java\javapath;C:\ProgramData\Oracle\Java\javapath\java.exe;C:\Program Files (x86)\Windows Live\Shared;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Users\cnnr2\Desktop;
USERNAME=cnnr2
OS=Windows_NT
PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 23 Stepping 10, GenuineIntel



---------------  S Y S T E M  ---------------

OS: Windows 10.0 , 64 bit Build 10240 (10.0.10240.16384)

CPU:total 4 (4 cores per cpu, 1 threads per core) family 6 model 23 stepping 10, cmov, cx8, fxsr, mmx, sse, sse2, sse3, ssse3, sse4.1, tsc

Memory: 4k page, physical 8387700k(4758692k free), swap 9698420k(4862576k free)

vm_info: Java HotSpot(TM) Client VM (25.66-b18) for windows-x86 JRE (1.8.0_66-b18), built on Nov  9 2015 10:58:29 by "java_re" with MS VC++ 10.0 (VS2010)

time: Sun Dec 20 20:41:12 2015
elapsed time: 74 seconds (0d 0h 1m 14s)

```

**Comment by ryanlarkin** [comment]
Ok, definitely relates to #71, but not duplicate as this is a crash, and the other is still semi-usable.

**Comment by ryanlarkin** [comment]
Fixed in #91
